wrappers/_shared.nix

```diff
@@ -1,37 +1,59 @@
-helpers:
+{
+  helpers,
+  filesOpt ? null,
+  filesPrefix ? "nvim/",
+  initName ? "init.lua",
+}:
```

Comment:
Maybe documenting those (inline) would be helpful.
Especially `filesOpt`.

Reply:
Sure, done